core/src/main/scala/akka/kafka/scaladsl/MetadataClient.scala

```diff
@@ -114,7 +99,9 @@ object MetadataClient {
       consumerSettings: ConsumerSettings[K, V],
       timeout: Timeout
   )(implicit system: ActorSystem, ec: ExecutionContext): MetadataClient = {
-    val consumerActor = system.actorOf(KafkaConsumerActor.props(consumerSettings))
+    val consumerActor = system
+      .asInstanceOf[ExtendedActorSystem]
+      .systemActorOf(KafkaConsumerActor.props(consumerSettings), "alpakka-kafka-metadata-client")
```

Comment:
A second call would fail because name taken. So it either needs an atomiclong or such to give it a unique id, or use an extension to keep it one shared actor per system

Reply:
Ah, right.  I'll use an atomic long.